apps/api/views.py

```diff
@@ -61,6 +65,26 @@
 log = commonware.log.getLogger('z.api')
 
 
+def auth_required(fn=None, permission=None):
+    def decorator(fn):
+        @wraps(fn)
+        def wrapper(request, *args, **kw):
+            if not (AMOOAuthAuthentication(two_legged=True)
```

Comment:
No. The Piston APIs use almost entirely different auth code from the DRF APIs. If we do wind up switching to three legged auth, it will hopefully be after we switch to DRF, so this should be gone.

This code is mainly moved from below, because I needed the same logic elsewhere and didn't want to repeat it.